message LocationReading {
    required double lattitude_geodetic_degrees = 1 [default = -9999];
    required double longitude_degrees = 2 [default = -9999];
    required double altitude_above_ellipsoid_km = 3 [default = -9999];
    required fixed64 timestamp_ms = 4 [default = 0];
}